### PR TITLE
Details about SushiSwap Onsen Menu

### LINKED ISSUE #1044: Updates to DEX Liquidity page
The existing liquidity program with Sushi is ending and we'll be transferring it to the Onsen program with them.

Therefore we need to amend the token front end to reflect this. 

Proposal is on the existing DEX Liquidity page: 
1. Remove everything 
2. Put in a strapline saying "Liquidity rewards have now been moved to the Sushi Onsen Menu <https://app.sushi.com/farm> with supporting documentation here <https://docs.sushi.com/products/yield-farming/what-is-onsen> and here <https://docs.sushi.com/faq-1/onsen-faq>"

I don't know if there's any value in putting historic information from the old pools there, my suspicion is there isn't any unless you think differently.

Thanks

### PR DESCRIPTION
**Pull request by campbellssource**
Screenshot 2021-12-18 at 14 24 23

Adds details about the new liquidity incentive program, keeps the old one for people who still have tokens in the pool.

closes #1044

### CODE REVIEW

File: src/i18n/translations/dev.json [hunk 1]
```diff
@@ -351,6 +350,11 @@
   "liquidityTokensContractTitle": "SLP Tokens earning rewards",
   "liquidityTotalAvailableRewards": "Total available rewards",
   "liquidityRewardsTitle": "Active liquidity rewards",
+  "liquidityOnsenIntro": "Liquidity rewards have been moved to the Sushi Onsen Menu. ",
```
Comment (candida-d) [inline src/i18n/translations/dev.json:353]:
```suggestion
  "liquidityOnsenIntro": "Earn rewards for providing liquidity on the SushiSwap Onsen menu.",
```

File: src/routes/liquidity/liquidity-container.tsx [hunk 2]
```diff
@@ -13,23 +13,26 @@ export const LiquidityContainer = ({ state }: { state: LiquidityState }) => {
   const { ethAddress } = useWeb3();
   return (
     <section className="liquidity-container">
-      <Callout icon={<Error />} intent="error" title={t("lpEndedTitle")}>
+      <h2>{t("liquidityRewardsTitle")}</h2>
+      <p>
+        {t("liquidityOnsenIntro")}
+        <a href={Links.SUSHI_ONSEN_WHAT_IS}>{t("liquidityOnsenHowItWorks")}</a>
```
Comment (candida-d) [inline src/routes/liquidity/liquidity-container.tsx:19]: Could these links be in list form, or some other way to format "how it works" and "FAQ"? That is - if they're formatted like the screenshot, and they could use some breathing room in there

Comment (campbellssource) [reply src/routes/liquidity/liquidity-container.tsx]: Done and updated screenshot

**Comment by candida-d** [comment]
This is now ready to go - it's live on Sushi.

**Review by dexturr — APPROVED** [review]

**Pull request merged**